src/commands/BirthdayCommand.ts

```diff
@@ -66,7 +66,10 @@ class BirthdayCommand extends Command {
 					.addBooleanOption((opt) => opt.setName("delete").setDescription("Delete your birthday entry."))
 			)
 			.addSubcommand((option) =>
-				option.setName("my").setDescription("Show what date is stored for your birthday.")
+				option
+					.setName("get")
+					.setDescription("Show what date is stored for a members birthday.")
+					.addUserOption((opt) => opt.setName("user").setDescription("The user to get the birthday of."))
```

Comment:
I'd make this optional and just use the current user if not specified.

Reply:
yeah good idea